plugins/woocommerce-admin/client/products/fills/details-section/details-section-fills.tsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import {
+	__experimentalWooProductSectionItem as WooProductSectionItem,
+	__experimentalWooProductFieldItem as WooProductFieldItem,
+	__experimentalProductFieldSection as ProductFieldSection,
+	useFormContext,
+} from '@woocommerce/components';
+import { registerPlugin } from '@wordpress/plugins';
+import { useState } from '@wordpress/element';
+import {
+	PRODUCTS_STORE_NAME,
+	WCDataSelector,
+	Product,
+} from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import { EditProductLinkModal } from '../../shared/edit-product-link-modal';
+import {
+	DetailsNameField,
+	DetailsCategoriesField,
+	DetailsFeatureField,
+	DetailsSummaryField,
+	DetailsDescriptionField,
+	DETAILS_SECTION_ID,
+} from './index';
+import './product-details-section.scss';
+
+const DetailsSection = () => {
+	const { values } = useFormContext< Product >();
+	const [ showProductLinkEditModal, setShowProductLinkEditModal ] =
+		useState( false );
+	const { permalinkPrefix, permalinkSuffix } = useSelect(
+		( select: WCDataSelector ) => {
+			const { getPermalinkParts } = select( PRODUCTS_STORE_NAME );
+			if ( values.id ) {
+				const parts = getPermalinkParts( values.id );
+				return {
+					permalinkPrefix: parts?.prefix,
+					permalinkSuffix: parts?.suffix,
+				};
+			}
+			return {};
+		}
+	);
+
+	return (
+		<>
+			<WooProductSectionItem
+				id={ DETAILS_SECTION_ID }
+				location="tab/general"
+				pluginId="core"
+			>
+				<ProductFieldSection
+					id="general/details"
+					title={ __( 'Product details', 'woocommerce' ) }
+					description={ __(
+						'This info will be displayed on the product page, category pages, social media, and search results.',
+						'woocommerce'
+					) }
+				/>
+			</WooProductSectionItem>
+			<WooProductFieldItem
+				id="details/name"
+				section={ DETAILS_SECTION_ID }
+				pluginId="core"
```

Comment:
Once you change the `default` order of the `FieldItem` and `SectionItem`, I would actually add the order to each field here.
We may want to separate each field by two, so give `details/name` order `1` the categories field order `3` etc...
With this approach I think it would be better to have a higher default like 20 for the slot fill wrappers.

Reply:
Right, feels a little funny but I see the logic. I've added these props in https://github.com/woocommerce/woocommerce/pull/36368/commits/b12fc103a3a4a34c363945c4c8c3fd2073090cc2.